subprojects/packagefiles/fmt/meson.build

```diff
@@ -13,7 +13,9 @@ fmt_lib = library(
   'src/os.cc',
   cpp_args: fmt_private_cpp_args,
   include_directories: 'include',
-  install: true,
+  build_by_default: false,
+  # so we do not build libfmt when using fmt_header_only
+  install: not meson.is_subproject(),
```

Comment:
This breaks things for people who use this wrap and do want to build and use fmt as a shared library.

The common solution in these cases is to have a project option to select whether the dep should be built as a lib or is used as a header only lib.